squash_bot/list_timetable/commands.py

```diff
@@ -2,13 +2,12 @@
 import enum
 import typing
 
-from . import timetable
-
 from squash_bot.core import command as _command
 from squash_bot.core import command_registry, lambda_function
 from squash_bot.core.command import CommandVerificationError
 from squash_bot.core.data import dataclasses as core_dataclasses
 
+from . import timetable
```

Comment:
ooc: is there a standard that shows this anywhere or is it more personal pref?

Reply:
Yep it's [here](https://github.com/AledWatkins/squash-bot/blob/83472f6049af5a0b65d733ab7eaf277cefb9bfca/pyproject.toml#L63-L68) -- ruff reorders them automatically